script/etcd/write-simple.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
```

Comment:
I was testing out whether or not reads from etcd were working. Wanted to test it against a known set of written data.